website/docs/developer/deployment-guide/deployment.md

```diff
@@ -1,8 +1,51 @@
 ---
 id: deployment
-title: Developer Deployment
+title: Deployment Information
 description: Provides developer information for ORCA code deployment.
 ---
 
-This document is currently under construction.
+## Introduction to ORCA
```

Comment:
Looks like this was copied from orca-intro. Don't think it's needed here.

Reply:
Removed this section.